### DISCUSSION TITLE
Rest path gets printed with 3 '/'s when camel rest component is 'platform-http-main'

### DISCUSSION THREAD

**Discussion by MN-Dheeraj** [apache/camel-karavan#1266] [General]
I created a route that takes in request through REST component as follows on karavan.

- route:
    id: route-b2e9
    description: Customer Demographic
    nodePrefixId: route-6d2
    from:
      id: from-c2dd
      uri: rest
      parameters:
        method: post
        path: /CustomerDemographic
        consumes: application/xml
When I make call to this url from postman - http://localhost:8080/CustomerDemographic and read endpoint to log to my DB by this line of code :
String fromUri = exchange.getFromEndpoint().getEndpointBaseUri();

I was getting "platform-http:///CustomerDemographic". I know karavan uses platform-http, but I don't know why I see 3 '/'s in the url instead of one.

When I use servlet as the rest component in my camel-spring boot code, I see it correctly -> servlet:/CustomerDemographic.
But, It is different on karavan.

Best Regards,
MN. Dheeraj

**Comment by davsclaus** [comment]
duplicate and closed as not a karavan user question

**Reply by MN-Dheeraj** [reply]
Please guide me to a proper channel to have this discussion and don't close it.

**Reply by davsclaus** [reply]
https://camel.apache.org/community/
